Assets/Mapzen/VectorData/Filters/PropertyValueFeatureMatcher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Mapzen.VectorData.Filters
+{
+    public class PropertyValueFeatureMatcher : PropertyFeatureMatcher
+    {
+        public List<object> ValueSet { get; set; }
+
+        protected override bool MatchesProperty(object property)
+        {
+            return ValueSet.Contains(property);
```

Comment:
Yeah totally, I've realized we're going to need a stricter representation for feature properties than just `object`. I imagine this could use a "variant" type which is either a string, a boolean, or a double. 

This is kind of tricky to solve now, because `ValueSet` may not even contain objects of a single type!